### ISSUE TITLE
screen share permission call twice.

### ISSUE THREAD

**Issue by theMasix** [0x006F/react-media-recorder#25]
Hello. I wanna use the screen capture alongside audio. So it's my code:
```
<ReactMediaRecorder
                screen
                render={({
                  status, startRecording, stopRecording, mediaBlobUrl,
                }) => ((
                  <div>
                    <p>{status}</p>
                    <button type="button" onClick={startRecording}>Start Recording</button>
                    <button type="button" onClick={stopRecording}>Stop Recording</button>
                    <video src={mediaBlobUrl} controls autoPlay loop />
                  </div>
                ))}
 />
```
but the screen share permission call twice. one after render <ReactMediaRecorder> and one after calling the startRecording.
It is a BUG i think...

**Comment by 0x006F** [comment]
The `getDisplayMedia()` behaves a bit differently than `getUserMedia()`. The granted permission won't persist. [You can read more here about it (from MDN) here, under the Usage Notes](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getDisplayMedia). So, each time you want to record the screen, the browser will _explicitly_ ask for the permission.

**Comment by 0x006F** [comment]
On a second read, you mentioned it was asking for permissions on 'render'. On my end, it didn't ask. (I used the same example you provided).

**Comment by theMasix** [comment]
It was an URGENT situation so I rewrite the whole package on my own. I think I should capture my screen to show you how it behaves. As i said, the screen capture permission calls twice and It's a bug.
I will send the video for you.
for now, my chrome version is: ```85.0.4183.83```

**Comment by 0x006F** [comment]
This should be fixed in #57 . New version `1.5.0`
